understanding/22/focus-not-obscured-enhanced.html

```diff
@@ -59,6 +59,7 @@ <h3>Additional Techniques (Advisory)</h3>
     <h3>Failures</h3>
     <ul>
       <li>An interaction that causes content to appear over the component with keyboard focus, visually covering part of the focus indicator. This behavior might be encountered with advertising or promotional material meant to provide more information about a product as the user navigates through a catalogue.</li>
+      <li>A light box or other semi-opaque effect that overlaps the focus indicator.  The obscuring may triggered by a component receiving focus or from other interation, but is a failure regardless.</li>
```

Comment:
@bruce-usab for some reason I cannot make suggestions in this change. I suggest the following:

> A light box or other semi-opaque effect that diminishes the visibility of the focus indicator. This form of  obscuring may be triggered by a component receiving focus or from another interaction, but where the item receiving focus is partially obscured due to a semi-opaque overlay, it is a failure regardless.

Reply:
Withdrawing this suggestion. Please see my comments in the Conversation. I think the lightbox language should be adjusted.